gssapi/raw/ext_cred_store.pyx

```diff
@@ -82,8 +83,16 @@ cdef gss_key_value_set_desc* c_create_key_value_set(dict values) except NULL:
                           "key-value set elements")
 
     for (i, (k, v)) in enumerate(values.items()):
-        res.elements[i].key = k
-        res.elements[i].value = v
+        if type(k) is str:
```

Comment:
Use `isinstance` is a better way of checking the type. By using `isinstance` it automatically handles subclassed string instances.

```suggestion
        if isinstance(k, str):
```

Reply:
I will change both `k` and `v` for that.